src/vehicle.cpp

```diff
@@ -654,6 +666,21 @@ void vehicle::control_engines() {
     start_engines();
 }
 
+int vehicle::select_boiler() {
+    uimenu tmenu;
+    std::string boiler_name;
```

Comment:
This variable is not needed outside the loop, it should therefor be defined inside the loop. And there it could be const.